### ISSUE TITLE
Charts with min/max no longer work in master-head

### ISSUE THREAD

**Issue by rmweir** [rancher/rancher#28331]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
1. try to deploy eks cluster or other system chart with a min or max

**Result:**
logs contain `template system-library-rancher-eks-operator allowed rancher version not match current server, requeuing`

**Other details that may be helpful:**
This is happening because of this server version format has changed for master-head: template system-library-rancher-eks-operator allowed rancher version not match current server, requeuing

min/max fields in questions.yaml are only ignore if the server version is "master", "dev", "", or has the suffix "head". The server version is now of the following format does not match any of them: https://github.com/rancher/rancher/commit/e2de9d4192ab227c2530916472b7fc512342bcb8

**Comment by sowmyav27** [comment]
**On latest master-head - commit id: `28ea6d4b1`**

- HA setup, run a CIS scan on a user cluster
- Scan is stuck in "Creating"
- Error seen in rancher logs

```
2020/08/11 16:15:52 [INFO] cisScanHandler: Create: deploying helm chart
2020/08/11 16:15:52 [ERROR] error syncing 'c-6grq7/cis-1597162532055677189': handler cisScanHandler: cisScanHandler: Create: error deploying app: cisScanHandler: deployApp: failed to find cis system catalog "": template system-library-rancher-cis-benchmark allowed rancher version not match current server, requeuin
```

**Comment by izaac** [comment]
`master-head (08/11/2020)` _28ea6d4b1_

- HA setup k8s
- Single node cluster DO and local HA.

Ember UI is stuck and can't continue install.

![Screen Shot 2020-08-11 at 10 30 37 AM](https://user-images.githubusercontent.com/27659/89929314-efff2880-dbbd-11ea-8a62-3c7678414d0a.png)

**Comment by jiaqiluo** [comment]
`master-28ea6d4b1bfd5369f9d4d4454256a5537ed3996d`

the available versions of monitoring show in the dropdown list contain only 0.0.1 - 0.0.3, but the latest version is 0.1.3

![image](https://user-images.githubusercontent.com/6218999/89930223-415be780-dbbf-11ea-9a02-d3d456f884ac.png)

**Comment by izaac** [comment]
`master-head (08/11/2020)` _28ea6d4b1_

Logging.v1

Logging chart gets deployed but the fluentd version installed is `0.1.17` I was expecting `0.1.19` as per https://github.com/rancher/rancher/issues/26543#issuecomment-666762821.

The deployed workload has config issues:

```
[error]: config error file="/fluentd/etc/config/entry/fluent.conf" error_class=Fluent::ConfigError error="Unknown parser plugin 'multi_format'. Run 'gem search -rd fluent-plugin' to find plugins"
```

**Comment by izaac** [comment]
`master-head (08/11/2020)` _28ea6d4b1_

cluster-alerting

```
Failed to install app cluster-alerting. Error: validation failed: [unable to recognize "": no matches for kind "Alertmanager" in version "monitoring.coreos.com/v1", unable to recognize "": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"]
```

Chart app versions shown in UI: 0.0.1, 0.0.2, 0.0.3. With `0.0.3` selected by default

**Comment by izaac** [comment]
Rancher version `master-head (08/11/2020)` _44caa5889_

- Logging.v1 works, fluentd v0.1.19 is installed and the app version is 0.2.1
- Alerting is also working App version is 0.1.3

**Comment by sowmyav27** [comment]
**Verified on master-head - commit id: `d19f179c4`**

- User is able to run CIS scan on an RKE cluster. Scan report is generated.

**Comment by jiaqiluo** [comment]
The bug fix is validated on `master-d19f179c4ff15bb782e27ed50d33d169dad79f80-head`

- provision an EC2 cluster
- go to the cluster page -> monitoring 

results:
- all available versions of the chart are in the dropdown list
- enabling monitoring with the latest chart succeeds 
- monitoring apps are deployed and work as expected
